logger.go

```diff
@@ -128,6 +130,16 @@ func New(options ...Option) *Logger {
 	res.callerOn = strings.Contains(res.format, "{{.Caller") || res.callerFile || res.callerFunc || res.callerPkg
 	res.levelBracesOn = strings.Contains(res.format, "[{{.Level}}]") || res.levelBraces
 
+	outFile, outOk := res.stdout.(*os.File)
+	errFile, errOk := res.stderr.(*os.File)
+	if outOk && errOk {
+		outStat, _ := outFile.Stat()
+		errStat, _ := errFile.Stat()
+		res.sameStream = os.SameFile(outStat, errStat)
```

Comment:
omitting error check bothers me here. I'm not sure in what situation Stat would fail (maybe permission error or smth like this), but Stat returns error and I think it is better to check

maybe it is also worth to extract the code to some isStreamsSame function as it's getting to big

Reply:
What we gonna do if Stat fail? I think to throw panic, but not sure.

Reply:
panics from the library is not a good idea. just returns safe default, i.e. false

Reply:
fixed. Added error handling, extract to function and make some tests. Because I added new import, number of lines changed and some tests failed, so i had to change them